blockchain/blockchain.go

```diff
@@ -34,6 +35,7 @@ type Reader interface {
 	Receipt(hash *felt.Felt) (receipt *core.TransactionReceipt, blockHash *felt.Felt, blockNumber uint64, err error)
 	StateUpdateByNumber(number uint64) (update *core.StateUpdate, err error)
 	StateUpdateByHash(hash *felt.Felt) (update *core.StateUpdate, err error)
+	L1HandlerTxnHash(msgHash *common.Hash) (l1HandlerTxnHashes *felt.Felt, err error)
```

Comment:
`l1HandlerTxnHashes` - there is only one hash right? should it be `l1HandlerTxnHash` ?

Reply:
Yup!